extensions/wikia/Helios/User.class.php

```diff
@@ -450,4 +460,9 @@ private static function clearCookie( $cookieName ) {
 			\WebResponse::NO_COOKIE_PREFIX
 		);
 	}
+
+	private static function getHeliosClient() {
+		global $wgHeliosBaseUri, $wgHeliosClientId, $wgHeliosClientSecret;
+		return new Client( $wgHeliosBaseUri, $wgHeliosClientId, $wgHeliosClientSecret );
+	}
```

Comment:
Can we use [dependency injection for this](https://github.com/Wikia/app/tree/dev/lib/Wikia/src/DependencyInjection)?